### DISCUSSION TITLE
Button usage

### DISCUSSION THREAD

**Discussion by DevBadger10** [gottadiveintopython/asynctkinter#4] [Q&A]
How could I make this trigger when a button is pressed?

**Comment by gottadiveintopython** [comment]
I don't understand the question.
Can you explain it with python code?

**Reply by DevBadger10** [reply]
Looking back it wasn't clear. I've found a fix that doesn't use TKinter but essentially: 
How would I use this to bind an event to a button press? But gtg sleep

**Reply by gottadiveintopython** [reply]
```python
button_instance.bind("<Button>", lambda event: asynctkitner.start(async_function(event)), "+")
```
